docs/links/community_portal.py

```diff
@@ -1,7 +1,7 @@
-from . import link
-
-link_name = "Community Portal" 
-link_text = "Community Portal" 
-link_url = "https://community.mautic.org" 
-
-link.xref_links.update({link_name: (link_text, link_url)})
+from . import link
+
+link_name = "Community Portal" 
+link_text = "Community Portal" 
+link_url = "https://community.mautic.org" 

```

Comment:
Please use the command `make link` at the terminal, follow the instructions, and create a link which specifically goes to the integrations proposals page: https://community.mautic.org/assemblies/marketing-team/f/53/. That will be much better than just dropping them at the Community Portal, and this link is likely to be already used in other places.

See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-the-external-links-extension for how this works.